### PR TITLE
Router resolve

### PR DESCRIPTION
**Pull request by roopkt**
Worked on creating ROUTER_RESOLVE_END action for the issue #358

### CODE REVIEW

**Comment by roopkt** [comment]
Finished creating ROUTER_RESOLVE_END action dispatching. Fixes #358

File: modules/store-devtools/src/config.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { ActionReducer } from '@ngrx/store';
+import { ActionReducer, Action } from '@ngrx/store';
```
Comment (royling) [inline modules/store-devtools/src/config.ts]: Is this an unused import?

Comment (roopkt) [reply modules/store-devtools/src/config.ts]: yes and I am doing another checkin to remove those unused namespaces.
Below are required only. Basically this file does not need to be changed. 
import { InjectionToken } from '@angular/core';
import { ActionReducer } from '@ngrx/store';

Comment (roopkt) [reply modules/store-devtools/src/config.ts]: Please undo any changes to this file. This is my mistake.

**Comment by roopkt** [comment]
@royling  @brandonroberts  Hi I resolved the merge issues and also removed the redundant namespace all checked in.

**Comment by roopkt** [comment]
Hi @brandonroberts  is there any plan to review my pull request ?

**Comment by roopkt** [comment]
Hi @brandonroberts 
Any idea when this PR will be reviewed ?

**Comment by brandonroberts** [comment]
Left a comment here about this feature: https://github.com/ngrx/platform/issues/358#issuecomment-344328228

**Pull request closed without merging**